README.md

```diff
@@ -235,6 +223,39 @@ target_link_libraries(
 To make the installed stdlib project discoverable add the stdlib directory to the ``CMAKE_PREFIX_PATH``.
 The usual install location of the package files is ``$PREFIX/lib/cmake/fortran_stdlib``.
 
+### Using stdlib with fpm
+
+To use `stdlib` within your `fpm` project, add the following lines to your `fpm.toml` file:
+```toml
+[dependencies]
+stdlib = { git="https://github.com/fortran-lang/stdlib", branch="stdlib-fpm" }
+```
+
+> **Warning**
+> 
+> Fpm 0.9.0 and later implements stdlib as a *metapackage*.
+> To include the standard library metapackage, change the dependency to:
+> `stdlib = "*"`.
+> 
+> [see also](https://fpm.fortran-lang.org/spec/metapackages.html)
+
+### Using stdlib with a regular Makefile
+
+After the library has been built, it can be included in a regular Makefile.
+If the library has been installed in a directory inside the compiler's search path,
+only a flag `-lfortran_stdlib` is required to link against `libfortran_stdlib.a` or `libfortran_stdlib.so`.
+If the installation directory is not in the compiler's search path, one can add for example
+```make
+libdir=${install_dir}/lib
+moduledir=${install_dir}/include/fortran_stdlib/<compiler name and version>
+```
+Here it is assumed that the compiler will look for libraries in `libdir` and for module files in `moduledir`.
+The library can also be included from a build directory without installation:
+```make
+libdir=$(build_dir)/src
+moduledir=$(build_dir)/src/mod_files
+```
+
```

Comment:
Should we add a something like below, to cover the case of compiling in a shell script or naively from the command line? I don't have much expertise on build systems, and often write small scripts to build programs.

### Using stdlib when compiling within a script, or on the command line
This example uses [pkg-config](https://www.freedesktop.org/wiki/Software/pkg-config/) to manage the compiler flags.
```
export PKG_CONFIG_PATH=${install_dir}/lib/pkgconfig:$PKG_CONFIG_PATH
export STDLIB_CFLAGS=$(pkg-config --cflags fortran_stdlib)
export STDLIB_LIBS=$(pkg-config --libs fortran_stdlib)

export FC=gfortran
export FFLAGS="-O3 -flto"

# Compile
$FC $FFLAGS -c $STDLIB_CFLAGS my_program.f90
# Link
$FC $FFLAGS -o my_program my_program.o $STDLIB_LIBS 
```

Reply:
I think the information on pkg-config would certainly be nice to include as well, I can do that. I am a little hesitant to include actual compile rules, since these will depend on the particular compiler used and perhaps also on other things. To explain this clearly to people who are not familiar with Makefiles and compilation rules could take too much space. Instead, it could be nice to include a few worked-out usage examples with Makefiles in a separate location.